src/transport/SecureSessionMgr.h

```diff
@@ -217,6 +217,12 @@ class DLL_EXPORT SecureSessionMgr : public TransportMgrDelegate
         kInitialized, /**< State when the object is ready connect to other peers. */
     };
 
+    enum class EncryptionState
+    {
+        kEncrypted,
```

Comment:
I wonder if we should name these as 'AlreadyEncrypted' or 'PerformEncryption'. When I read:

```
SendMessage(...., EncryptionState::kEncrypted)
```

I wonder which of these 2 is correct:
   - perform encryption since parameter says 'use encryption' *OR*
   - do not perform encryption, since parameter says 'state is already encrypted'

Reply:
`kEncrypted` sounds like the state of the current encryption, rather than an action that needs to be performed on the packet. However, I am happy to rename it to make it more clear.

How about changing it to this?
```
    enum class EncryptionState
    {
        kPayloadIsEncrypted,
        kPayloadIsUnencrypted,
    };
```

Reply:
Updated the enum variants..